packages/interbit-template/package.json

```diff
@@ -29,12 +29,12 @@
         "redux-saga-test-plan": "^3.6.0"
     },
     "scripts": {
-        "start": "cross-env REACT_APP_INTERBIT_WEB_SERVER=123 REACT_APP_INTERBIT_PORT=5000 react-scripts start",
```

Comment:
Changing the port here is not recommended unless you change the corresponding port configuration in app-account and update the network's peers list in each app.